greenwave/consumers/consumer.py

```diff
@@ -226,7 +194,5 @@ def _publish_decision_change(
                 decision['testcase'] = testcase
 
             log.info('Publishing a decision update message: %r', decision)
-            if self.flask_app.config['MESSAGING'] == 'fedmsg':
-                self._publish_decision_update_fedmsg(decision)
-            elif self.flask_app.config['MESSAGING'] == 'fedora-messaging':
+            if self.flask_app.config['MESSAGING'] == 'fedora-messaging':
```

Comment:
Can we drop the `MESSAGING` option completely?

Reply:
ok. we only have one option now, we don't need to think about whether we will have more options in the future. fixed